app/process-batches/reprocess-if-needed.js

```diff
@@ -50,7 +51,7 @@ async function success (filename) {
 
 async function failed (filename) {
   console.log('Previous processing failure status set, quarantining')
-  await blobStorage.quarantinePaymentFile(filename, filename)
+  await quarantineFile(filename)
```

Comment:
Edi has picked up in QA that becuz `quarantineFile` logs out too there is what seems lik duplicating logging statements -- should be remove these log statements and relay on `quarantineFile` error statements?

Reply:
I don't think so because this one provides more context as to why.  ie this one says we've already failed to process this batch so time quarantine.

Whereas the `quarantineFile` is more a generic "I'm quarantining this now".

If we were going to remove one I'd remove the one in `quarantineFile` as it adds no additional context.